### ISSUE TITLE
Settings matching issue - possible DRM matching error?

### ISSUE THREAD

**Issue by Lexiconmancer** [IsThereAnyDeal/IsThereAnyDeal-Issues#3124]
Weird West is currently -50% on Epic Game Store.

My settings are to notify for games with
-any price
-price cut at least 30%
-with Steam DRM /or/ no DRM
-any store

However when I look at my waitlist it says this game does not match my settings. Since other games are properly notifying with a 30% or greater price cut, and other games on Epic Game Store which match my settings are not notifying, I suspect this is a DRM issue. However, the notification limit settings do not have an option for "Epic Game Store" DRM. I am reasonably certain after doing some research that Weird West is not using UPlay or Origin DRMs (which I have disabled.) I don't have the option to select any other DRM options.

So in summary: Epic Game Store deals are not being counted as matching my notification settings, despite matching my price settings and when there is no known conflict with current DRM settings. I'm not entirely sure why, but it may be DRM-related.

Thanks!
